package.json

```diff
@@ -20,7 +20,11 @@
   },
   "homepage": "https://github.com/Camilovelag/Webpack_demo#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
     "html-webpack-plugin": "^5.5.0",
     "style-loader": "^3.3.1",
     "webpack": "^5.74.0",
```

Comment:
- [ ] I will suggest you to use the following scripts:
```json
{
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
}
```

Reply:
Done! 😄 
https://github.com/Camilovelag/Leaderboard/blob/7d09f602259723ef9445ab6a89463c1d72b9746a/package.json#L6-L12